tests/e2e/test_lambda_handler.py

```
@@ -0,0 +1,67 @@
+import json
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)